### PR TITLE
Wrap errors using '%w' in the format strings

### PR DESCRIPTION
**Pull request by swetharepakula**
* log lines should use '%s' or '%q'

/assign @prameshj

### CODE REVIEW

File: pkg/psc/controller.go [hunk 1]
```diff
@@ -345,7 +345,7 @@ func (c *Controller) garbageCollectServiceAttachments() {
 		}
 		key, err := cache.MetaNamespaceKeyFunc(sa)
 		if err != nil {
-			klog.V(4).Infof("failed to generate key for service attachment: %s/%s: %w", sa.Namespace, sa.Name, err)
+			klog.V(4).Infof("failed to generate key for service attachment: %s/%s: %s", sa.Namespace, sa.Name, err)
```
Comment (prameshj) [inline pkg/psc/controller.go:348]: %q for consistency?

**Review by prameshj — COMMENTED** [review]
lgtm except one nit.

File: pkg/psc/controller.go (hunk 1 above)
Comment (swetharepakula) [reply pkg/psc/controller.go:348]: that sounds reasonable to me. thanks!

**Pull request merged**